### PR TITLE
Update recipe to `v0.1.3` (from PyPI)

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged) `NA`
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sugatoray** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
Thanks for taking a look @sugatoray!

It seems the LICENSE isn't packaged in the sdist so would need to be copied here:
```
ValueError: License file given in about/license_file (/home/conda/recipe_root/LICENSE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

image

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,20 +23,29 @@ requirements:
...
+    
+    PyPI: [https://pypi.org/project/opencensus-context/](https://pypi.org/project/opencensus-context/)
+    
   doc_url: https://github.com/census-instrumentation/opencensus-python/tree/master/context/opencensus-context/examples
   dev_url: https://github.com/census-instrumentation/opencensus-python/tree/master/context/opencensus-context
 
 extra:
   recipe-maintainers:
     - basnijholt
+    - sugatoray
```
Comment (dhirschfeld) [inline recipe/meta.yaml]:
```suggestion
  recipe-maintainers:
    - basnijholt
    - dhirschfeld
    - sugatoray
```

Comment (sugatoray) [reply recipe/meta.yaml]: @dhirschfeld Thank you for troubleshooting it. :rocket:

**Comment by sugatoray** [comment]
cc: @basnijholt

**Comment by basnijholt** [comment]
Still marked as draft, is that intentional?

**Comment by sugatoray** [comment]
> Still marked as draft, is that intentional?

Thanks. Forgot about the `draft` status! :+1:

**Comment by sugatoray** [comment]
@basnijholt Please have a look at the PR. It's no longer a draft now.

**Pull request merged**